lambda/lambda.js

```diff
@@ -99,6 +99,11 @@ async function handler(request) {
                     resolve(false);
                     return;
                 }
+                if (Math.floor(Date.now() / 1000) + 3600 < payload.exp) {
+                    console.log(jwtKey.modeName + ': \'Expiration time\' (exp) claim is too far in the future ');
```

Comment:
please remove the trailing whitespace. You may also use `"` to avoid escaping the `'`